canonical-web-and-design/jaas.ai
# Add slider content to the homepage hero section

## Done
Add slider content to the homepage hero section

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Click the four sections in the hero and see the content changes
- The content matched [the design](https://app.zeplin.io/project/5cf8ca699ee7bf1df9082012/screen/5d0baf121c726159faaaaf36)
